AdminWebsite/AdminWebsite/ClientApp/src/app/work-allocation/manage-team/manage-team.component.ts

```diff
@@ -1,7 +1,81 @@
-import { Component, OnInit } from '@angular/core';
+import { Component } from '@angular/core';
+import { faCircleExclamation, faExclamationCircle } from '@fortawesome/free-solid-svg-icons';
+import { FormBuilder, FormGroup } from '@angular/forms';
+import { JusticeUserResponse } from '../../services/clients/api-client';
+import { VideoHearingsService } from '../../services/video-hearings.service';
+import { Logger } from '../../services/logger';
 
 @Component({
     selector: 'app-manage-team',
-    templateUrl: './manage-team.component.html'
+    templateUrl: './manage-team.component.html',
+    styleUrls: ['./manage-team.component.scss']
 })
-export class ManageTeamComponent {}
+export class ManageTeamComponent {
+    private filterSize = 20;
+
+    constructor(private fb: FormBuilder, private videoHearingService: VideoHearingsService, private logger: Logger) {
+        this.form = fb.group({
+            inputSearch: ['']
+        });
+    }
+
+    loggerPrefix = '[ManageTeamComponent] -';
+    displayMessage = false;
+    faExclamation = faCircleExclamation;
+    faError = faExclamationCircle;
+    message: string;
+    users: JusticeUserResponse[];
+    form: FormGroup;
+    isEditing = false;
+    isSaving = false;
+    error = false;
+    displayAddButton = false;
+    errorMessage = false;
+
+    editUser(id) {
+        // TODO: edit will enable all the inputs in a row.
+        // const row = this.elRef.nativeElement.getElementsByClassName(id) as HTMLCollection;
+        // for (let i = 0; i < row.length; i++) {
+        //     (<HTMLElement>row[i]).removeAttribute('disabled');
+        // }
+    }
+
+    deleteUser(id) {}
```

Comment:
as per previous comment, think we can remove this for now as it is empty

Reply:
removed